README.md

```diff
@@ -16,10 +16,10 @@ Toolkit components that will simplify your iOS app development with ArcGIS Runti
 * [Bookmarks](Documentation/Bookmarks)
 
 ## Requirements
-* [ArcGIS Runtime SDK for iOS](https://developers.arcgis.com/en/ios/) 100.7.0 (or higher)
+* [ArcGIS Runtime SDK for iOS](https://developers.arcgis.com/en/ios/) 100.8.0 (or higher)
 * Xcode 10.2 (or higher)
```

Comment:
Conveniently, we're having a similar conversation over in the Tutorials repo. I'm not the authoritative source here but @nixta has [informed us](https://github.com/ArcGIS/DevLabs/pull/43#discussion_r405002119) that as of 100.8 we are dropping support for Xcode 10. You may want to figure out the details here and adjust this line.

Reply:
Ok, will do.  Thanks.

Reply:
I guess that means the toolkit can now take advantage of features [introduced](https://swift.org/blog/swift-5-1-released/) in Swift 5.1!